src/storage/btree.c

```diff
@@ -16506,7 +16506,11 @@ btree_find_min_or_max_key (THREAD_ENTRY * thread_p, BTID * btid, DB_VALUE * key,
 	}
     }
 
-  (void) pr_clone_value (value_p, key);
+  if (is_visible)
```

Comment:
`is_visiable` is updated in the while-block above. It only gets the last value of the btree_is_key_visible.

Reply:
ok, I understood.